4_arm.ipynb

```diff
@@ -644,6 +1149,19 @@
     "</div>"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "lang": "en"
+   },
+   "source": [
+    "<div class=\"alert alert-block alert-info\">\n",
+    "    <b>What's next</b>\n",
+    "    <p>In this notebook we have teached the robot about the object positions.</p>\n",
```

Comment:
"    <p>In this notebook we taught the robot about the object positions.</p>\n",